signtool/signtool.py

```diff
@@ -168,12 +179,13 @@ def parse_cmdln_opts(parser, cmdln_args):
         else:
             formats.append(fmt)
 
-    # bug 1164456
-    # GPG signing must happen last because it will be invalid if done prior to
-    # any format that modifies the file in-place.
-    if "gpg" in formats:
-        formats.remove("gpg")
-        formats.append("gpg")
+    # bug 1382882, 1164456
+    # Widevine and GPG signing must happen last because they will be invalid if
+    # done prior to any format that modifies the file in-place.
```

Comment:
I think we're ok. Signingscript is only sending one format at a time to signtool.